index.html

```diff
@@ -1,123 +1,114 @@
-
-      <p>Our documentation is <a href=
-      "https://pillow.readthedocs.io/">hosted on readthedocs.io</a>
-      and includes <a href=
-      "https://pillow.readthedocs.io/en/stable/installation.html">installation
-      instructions</a>, <a href=
-      "https://pillow.readthedocs.io/en/stable/handbook/index.html">handbook</a>,
-      <a href=
-      "https://pillow.readthedocs.io/en/stable/reference/index.html">API
-      reference</a>, <a href=
-      "https://pillow.readthedocs.io/en/stable/releasenotes/index.html">release
-      notes</a> and more.</p>
+    <hr>
 
-      <h3>Discussion</h3>
-      <p>Discussion occurs on <a href="https://github.com/python-pillow/Pillow/issues">GitHub</a>, <a href="https://stackoverflow.com/questions/tagged/python-imaging-library">Stack Overflow</a>, <a href="https://gitter.im/python-pillow/Pillow">Gitter</a> and <a href="irc://irc.freenode.net#pil">IRC</a>.</p>
-	    
-      <h3>For enterprise</h3>
-      <p>Available as part of the Tidelift Subscription.</p>
-      <p>
-      The maintainers of Pillow and thousands of other packages are working with Tidelift to deliver commercial support and maintenance for the open source dependencies you use to build your applications. Save time, reduce risk, and improve code health, while paying the maintainers of the exact dependencies you use.
-      <a href="https://tidelift.com/subscription/pkg/pypi-pillow?utm_source=pypi-pillow&utm_medium=referral&utm_campaign=enterprise&utm_term=repo">Learn more.</a>
-      </p>
-      
+  </div> <!-- /container -->
 
-      <h3>More</h3>
-      <p>Also check out <a href="https://python-pillow.org/pillow-perf">Pillow Perf</a></p>
-    </div><!-- FOOTER  -->
+</main>
 
-    <div id="footer_wrap" class="outer">
-      <div class="inner">
-        <p>&copy; Copyright 1995-2011, Fredrik Lundh, 2010-2020 <a href=
-        "https://github.com/python-pillow/Pillow/graphs/contributors">
-        Alex Clark and Contributors</a>. Published with <a href=
-        "https://github.com/python-pillow/python-pillow.github.io">GitHub
-        Pages</a>. Pillow logo <a href=
-        "https://github.com/python-pillow/Pillow/issues/575">created
-        by Alastair Houghton</a>. Random psychedelic art by
-        <a href="https://jeremykun.com/">Jeremy Kun</a>.</p>
-      </div>
-    </div>
-  </div>
-</body>
+<footer class="container">
+  <p>&copy; Copyright <a target="_blank" href="https://effbot.org/zone/copyright.htm">Fredrik Lundh (PIL author)</a>, <a target="_blank" href="https://aclark.net/#pillow">Alex Clark (Pillow fork author)</a> and <a target="_blank" href="https://github.com/python-pillow/Pillow/graphs/contributors">GitHub Contributors</a> 1995-2020. Logo by <a target="_blank" href="https://github.com/python-pillow/Pillow/issues/575">Alastair Houghton</a>. Psychedelic art by <a target="_blank" href="https://jeremykun.com/2012/01/01/random-psychedelic-art/">Jeremy Kun</a>.</p>
```

Comment:
We should probably change "GitHub Contributors" to "Contributors", as there have been some pre-fork and pre-GitHub, and I expect we've had some contribs not via GitHub post-fork too.

The link can remain.